### ISSUE TITLE
[BUG] corners not rounded

### ISSUE THREAD

**Issue by macjohnny** [bithost-gmbh/ngx-mat-select-search#176]
**Describe the bug**
see discussion in https://github.com/bithost-gmbh/ngx-mat-select-search/pull/169

**Screenshots**
![ngx-mat-select-search-rounded-corners](https://user-images.githubusercontent.com/5589029/66817681-66021700-ef3c-11e9-9a0f-9cfb9ed597fa.png)

Affected version: 1.8.0

**Comment by jfcere** [comment]
My temporary fix...

```sass
// temporary fix for missing round corners
// https://github.com/bithost-gmbh/ngx-mat-select-search/issues/176
ngx-mat-select-search ::ng-deep .mat-select-search-inner {
  border-radius: 4px 4px 0 0;
}
```

**Comment by macjohnny** [comment]
@jfcere would you like to submit your contribution as a PR?

**Comment by jfcere** [comment]
There ya go 😄

**Comment by macjohnny** [comment]
@jfcere thanks for the fix, it was released with version `2.1.0`
